### PR TITLE
Allow display and sort by stars

### PR DESCRIPTION
**Pull request by denis-yuen**
https://github.com/ga4gh/dockstore/issues/830

- [x] Check that you pass the basic compilation and unit tests by running `grunt`

### CODE REVIEW

File: app/styles/ds-style-fix.scss [hunk 1]
```diff
@@ -52,12 +52,15 @@ table.containers-grid {
...
   }
   th:nth-child(3) {
-    width: 20%;
+    width: 17.5%;
   }
   th:nth-child(4) {
+    width: 20%;
+  }
+  th:nth-child(5) {
     width: 37.5%;
```
Comment (garyluu) [inline app/styles/ds-style-fix.scss]: 25%+10%+17.5%+20%+37.5% = 110%

File: app/templates/containersgrid.html [hunk 2]
```diff
@@ -56,6 +63,9 @@
                 <a href="/containers/{{container.tool_path}}">{{container.name + (container.toolname ? '/' + container.toolname : '')}}</a>
               </td>
               <td>
+                {{ container.starredUsers.length == 0 ? '' : container.starredUsers.length }}
```
Comment (garyluu) [inline app/templates/containersgrid.html]: It should probably be either "container.starredUsers.length ? " or "container.starredUsers.length === 0 ? "

File: app/templates/workflowsgrid.html [hunk 3]
```diff
@@ -53,6 +60,9 @@
               <a ng-href="/workflows/{{workflow.path}}">{{workflow.repository + (workflow.workflowName ? ' (' + workflow.workflowName + ')' : '')}}</a>
             </td>
             <td>
+              {{ workflow.starredUsers.length == 0 ? '' : workflow.starredUsers.length }}
```
Comment (garyluu) [inline app/templates/workflowsgrid.html]: same here

File: app/styles/ds-style-fix.scss (hunk 1 above)
Comment (denis-yuen) [inline app/styles/ds-style-fix.scss]: Hah! Good point

**Comment by denis-yuen** [comment]
@garyluu  note, it looks like the UI was already sorting alphabetically by name when it should really be done by the web service.

**Pull request merged**